core/sequencer/runner/native_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	pb "github.com/google/keytransparency/core/api/v1/keytransparency_go_proto"
 )
 
+func fakeMetricFn(label string) {}
```

Comment:
Consider making this fake backed by a map. It would be a nice testing touch to assert that the metrics you will use for monitoring are actually being incremented as you expect.